addon/services/intl.js

```diff
@@ -175,7 +175,7 @@ export default Ember.Service.extend(Ember.Evented, {
         }
     },
 
-    getLanguage: function (locale) {
+    findLanguage: function (locale) {
```

Comment:
These are not yet documented and private, I am changing them to match what gets invoked on the adapter for consistency.  In 1.3.0 I plan to document this and mark it as public.